chaste_codegen/chaste_model.py

```diff
@@ -664,29 +649,15 @@ def _get_derivative_eqs_voltage(self):
                     derivatives.add(s)
         return eqs
 
-    def _get_derived_quant_annotated(self):
-        """ Get the variables annotated in the model as derived derived-quantity"""
-        return self._model.get_variables_by_rdf((self._PYCMLMETA, 'derived-quantity'), 'yes')
-
-    def _get_derived_quant_exposed(self):
-        """ Get the variables in the model that have exposed annotation and are derived quantities
-            (irrespective of any derived-quantity tags)"""
-        return [q for q in self._model.get_derived_quantities()
-                if self._model.has_ontology_annotation(q, self._OXMETA)] +\
-               [self._membrane_stimulus_current] if self._membrane_stimulus_current is not None else []
-
     def _get_derived_quant(self):
         """ Get all derived quantities
 
-            Note: the result depends on self.expose_annotated_variables to determine whether or not to include
-            variables exposed with oxford metadata that are derived quantities but are not annotated as such"""
-        if self.expose_annotated_variables:
-            # Combined and sorted in display name order
-            return \
-                sorted(set(self._get_derived_quant_annotated() + self._get_derived_quant_exposed()),
-                       key=lambda v: self._model.get_display_name(v, self._OXMETA))
-        else:
-            return self._get_derived_quant_annotated()  # These are already sorted
+        Stimulus currents are ignored and the result is sorted by display name"""
+        return sorted(set([q for q in self._model.get_derived_quantities() +
```

Comment:
This will only keep vars with an explicit derived-quantity annotation if they also have an oxmeta annotation?

Reply:
O yes uups